Week2/homework/maartjes_work.js

```diff
@@ -43,5 +43,12 @@ const tuesday = [
 ];
 
 const tasks = monday.concat(tuesday);
+const hourlyRate = 50;
 
 // Add your code here
+const totalEarnings = tasks
```

Comment:
Great job. For the next time you could also consider creating an arrow functions to which you pass on arguments. Something like this: 

const calculateEarnings = (arrayOfTasks, hourlyWage) =>
arrayOfTasks.map((task) => task.duration / 60)
.filter((hoursAmount) => hoursAmount >= 2)
.map((paidHoursAmount) => paidHoursAmount * hourlyWage)
.reduce((sum, amount) => sum + amount, 0)
.toFixed(2);

console.log(calculateEarnings(tasks, hourlyRate))